feat: add solid meta docs (#670)
solidjs/solid-docs-next: src/routes/solid-meta/reference/meta/metaprovider.mdx

---
title: MetaProvider
order: 6
---

> `MetaProvider` is a parent component responsible for wrapping all your meta components.

Reviewer: ```suggestion
`MetaProvider` is a parent component responsible for wrapping all the metadata components.
```
